alcs-frontend/src/app/features/application/overview/overview.component.ts

```diff
@@ -79,9 +80,6 @@ export class OverviewComponent implements OnInit, OnDestroy {
             this.decisionService.fetchByApplication(app.fileNumber).then((res) => {
               this.$decisions.next(res);
             });
-            this.reviewService.fetchReview(app.fileNumber).then((res) => {
-              this.$review.next(res);
-            });
```

Comment:
Is `reviewService` used elsewhere in the file or can we also remove its import and constructor?

Reply:
Good catch, removed